### DISCUSSION TITLE
Can't restore my backup file..

### DISCUSSION THREAD

**Discussion by xlemnlx** [lnreader/lnreader#728] [Q&A]
Hello. I would like to ask why I can't restore my backup. I've screenshot the error message. Hope the error message helps. 🙇‍♂️![Screenshot_20230806_180422_com.rajarsheechatterjee.LNReader.jpg](https://github.com/LNReader/lnreader/assets/20958309/b384bc35-9331-4c81-ba85-2f85bf4c5603)

![Screenshot_20230806_181137_com.huawei.filemanager.jpg](https://github.com/LNReader/lnreader/assets/20958309/c62f3ab4-a09d-4242-9f83-cdfaba553682)

**Comment by rajarsheechatterjee** [comment]
Duplicate of https://github.com/LNReader/lnreader/issues/633
